packages/features/calendars/DatePicker.tsx

```diff
@@ -138,6 +139,21 @@ const Days = ({
     ? availableDates(props.includedDates)
     : props.includedDates;
 
+  useEffect(() => {
+    if (selected && selected.date()) {
```

Comment:
We prefer early return https://handbook.cal.com/engineering/best-practices/prefer-early-returns